content/en/news/releases/1.9.x/announcing-1.9/_index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Announcing Istio 1.9
+linktitle: 1.9
+subtitle: Major Update
+description: Istio 1.9 release announcement.
+publishdate: 2021-02-09
+release: 1.9.0
+skip_list: true
+aliases:
+    - /news/announcing-1.9
+    - /news/announcing-1.9.0
+---
+
+We are pleased to announce the release of Istio 1.9!
+
+{{< tip >}}
+Officially supported versions of Kubernetes releases are 1.17 to 1.20 for this Istio release.
+{{< /tip >}}
+
+This is the first Istio release for 2021.  We would like to thank the entire Istio community, and especially the release
+managers `Shamsher Ansari` (Red Hat), `Steven Landlow` (Google) and `Jacob Delgado` (Aspen Mesh) for helping to get Istio 1.9.0
+published on schedule with quality.
+
+
+Our core focus for the 1.9 release was to improve the [Day 2 operations](https://dzone.com/articles/defining-day-2-operations)
+for users running Istio in production. Building upon the feedback collected from the User Experience Working
+Group, we wanted to improve the stability and overall upgrade experience for our users. A key aspect of
+stability is publishing accurate [feature status](https://istio.io/latest/about/feature-stages/) for Istio core APIs
+and features, and progressing their stability to enable our users to utilize Istio's capabilities with confidence,
+which was another focus for Istio 1.9 release.
+
+Our core focus for the 1.9 release was to improve the [Day 2 operations](/docs/reference/glossary/#D) for users
+running Istio in production. Building upon the feedback collected from the User Experience Working
+Group, we wanted to improve the stability and overall upgrade experience for our users. A key aspect of
+stability is publishing accurate [feature status](https://istio.io/latest/about/feature-stages/) for Istio core APIs
+and features, and progressing their stability to enable our users to utilize Istio's capabilities with confidence, which
+was another focus for Istio 1.9 release.
+
+Thank you to our users who participated in user experience surveys and empathy session to ensure Istio 1.9 is our most stable release to date.
+
+We will be publishing our plans to continue improving the Day 2 operations as a focus area for Istio 2021 roadmap in an upcoming blog .
+
+Here are some highlights for this release:
+
+## Feature promotion
```

Comment:
"New features."

Some are experimental, some are Beta, but the user doesn't really care what they _used_ to be, more what they are today.

Change headings to "Virtual Machine integration (Beta)" etc?

Reply:
Updated the headings based on your feedback.